internal/protocol/version.go

```diff
@@ -40,19 +42,19 @@ func (vn VersionNumber) String() string {
 	case VersionTLS:
 		return "TLS dev version (WIP)"
 	default:
+		if vn > 0x51303300 && vn <= 0x51303399 {
```

Comment:
Define 0x51303300 as const and use it for the other magic numbers.

Reply:
Done